lib/utils/github.ts

```diff
@@ -90,6 +90,10 @@ const isValidPullRequestUrl = (url: string): boolean => {
   return url.match(/((https?:\/\/)?(www\.)?github\.com\/[^\/]+\/[^\/]+\/pull\/[0-9]+)/) ? true : false;
 };
 
+const isValidIssueUrl = (url: string): boolean => {
+  return url.match(/((https?:\/\/)?(www\.)?github\.com\/[^\/]+\/[^\/]+\/issues\/[0-9]+)/) ? true : false;
+};
+
```

Comment:
You will also need to add test for this in https://github.com/open-sauced/insights/blob/beta/tests/lib/utils/github.test.ts

Reply:
Thanks for the heads up!